### ISSUE TITLE
Angle's eglSwapBuffers doesn't use vsync, but only in the UWP app

### ISSUE THREAD

**Issue by paulrouget** [servo/servo#25263]
On Windows, if I'm not mistaken, the Glutin port uses Angle. A swapbuffer take around ~15ms (good). In the UWP port, it takes < 1ms.

We expect swapbuffer to block, but it doesn't, eating all the CPU. Adding `eglSwapInterval(mEglDisplay, 1)` doesn't appear to help.

**Comment by jdm** [comment]
Is this something that changed recently, or just something you noticed for the first time? https://github.com/servo/servo/pull/25219 merged recently and included an ANGLE update.

**Comment by jdm** [comment]
On Windows, glutin only uses ANGLE if you run it the `--angle` argument.

**Comment by paulrouget** [comment]
Ok - I was wrong. Buffer swapping happens at a proper frame rate.

What's going on though is that we wake up the event loop way too often, and the buffer are not swapping 99% of the time because there's nothing to redraw.

Because we set the animation state to Animating and the the embedder event loop assume a redraw + buffer swapping, it should not go faster than 60 FPS. But if it's set to animating, and no redraw happens, no buffer swapping happens, so no blocking, so loop going full speed.

Closing. I'll see what we can do about the buggy animating state.
